src/Update/Nix/FetchGit/Utils.hs

```diff
@@ -41,15 +40,7 @@ ourParseNixFile :: FilePath -> IO (Either Warning NExprLoc)
 ourParseNixFile f =
   parseNixFileLoc f >>= \case
     Failure parseError -> pure $ Left (CouldNotParseInput parseError)
-    Success expr -> pure $ pure $ fixNixSets expr
-
--- Convert all NRecSet values (recursive sets) to NSet values because
--- we do not care about the distinction between NRecSet and NSet and
--- we want our program to treat both types equally.
-fixNixSets :: NExprLoc -> NExprLoc
-fixNixSets = transform fix
-  where fix (AnnE s (NRecSet bindings)) = AnnE s (NSet bindings)
-        fix n = n
+    Success expr -> pure <$> reduceExpr Nothing expr
```

Comment:
Just noticed this - we could remove `reduceExpr` as it does something different (resolves imports and reduces these into one expression) but it's probably harmless.